### PR TITLE
Bowtie2 viralrecon

### PR DESCRIPTION
**Pull request by JoseEspinosa**
Adding `bowtie2-build` and `bowtie2-align` as listed on this [issue](https://github.com/nf-core/viralrecon/issues/149)

### CODE REVIEW

File: software/bowtie2/build/meta.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+          type: file
+          description: Bowtie2 genome index files
+          pattern: "*.bt2"
+    - version:
+          type: file
+          description: File containing software version
+          pattern: "*.{version.txt}"
+authors:
+    - "@joseespinosa"
+    - "drpatelh"
```
Comment (drpatelh) [inline software/bowtie2/build/meta.yml:53]:
```suggestion
    - "@drpatelh"
```

**Review by drpatelh — APPROVED** [review]
👌🏽 😁

**Comment by drpatelh** [comment]
Ah, I forget you still need to add a `.github/workflows/bowtie2_build.yml` and `.github/workflows/bowtie2_align.yml` file for each of these. Sorry, would you mind adding that in a separate PR? Everything else looks cushty 👍🏽

**Comment by JoseEspinosa** [comment]
> Ah, I forget you still need to add a `.github/workflows/bowtie2_build.yml` and `.github/workflows/bowtie2_align.yml` file for each of these. Sorry, would you mind adding that in a separate PR? Everything else looks cushty 👍🏽

Yeah sure, on my previous PR of `samtools` I did it but this time I forgot it 😢

**Pull request merged**